### PR TITLE
Add drafts for the static pages

### PR DESCRIPTION
**Pull request by thomasleese**
This isn't the final content, but it adds the pages in advance of the content being ready.

### CODE REVIEW

File: app/views/layouts/base.html.erb [hunk 1]
```diff
@@ -41,7 +41,9 @@
     <%= govuk_footer(
       meta_items_title: 'Footer links',
       meta_items: {
+        'Accessibility': '/accessibility',
         'Cookies': '/cookies',
+        'Privacy': '/privacy',
```
Comment (tvararu) [inline app/views/layouts/base.html.erb:46]: I think we should comment the links out. We should be careful about making the service "look complete" when actually some of it is placeholder, as someone who isn't familiar might say "okay it's done" and we might ship it before it's ready.

File: config/locales/en.yml [hunk 2]
```diff
@@ -6,7 +6,8 @@ en:
 
   service:
     name: Apply for qualified teacher status (QTS) in England
-    email: my-service@email
+    email: qts.enquiries@education.gov.uk
```
Comment (tvararu) [inline config/locales/en.yml:9]: Is this the right email? Looks good, just would like to double check in Slack if you haven't already.

**Review by tvararu — APPROVED** [review]

File: app/views/layouts/base.html.erb (hunk 1 above)
Comment (thomasleese) [reply app/views/layouts/base.html.erb:46]: I figured the HTTP basic authentication prevented us from thinking the service looked complete, but happy to comment them out if you think that's worth it.

Comment (tvararu) [reply app/views/layouts/base.html.erb:46]: Maybe it's fine. Just keep it in mind come go-live that someone still has to do a pass for those. :)

**Pull request merged**